### PR TITLE
feat: Add data table

### PR DESCRIPTION
**Pull request by cwarren5701**

### CODE REVIEW

File: IntroToCSharp/Shared/Components/Activities/DataTable.razor [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+@code {
+    [Parameter]
+    public string[] Headers { get; set; } = {};
+    [Parameter]
+    public string[] Rows { get; set; } = {};
+
+    private string _headers => string.Join(", ", Headers);
+    private string _rows => string.Join(", ", Rows);
+
+
```
Comment (jcollard) [inline IntroToCSharp/Shared/Components/Activities/DataTable.razor:33]: Should not have more than 1 blank space in a row.

File: IntroToCSharp/Shared/Components/Activities/DataTable.razor [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
```
Comment (jcollard) [inline IntroToCSharp/Shared/Components/Activities/DataTable.razor:1]: Should not have a blank space on the first line.

**Review by jcollard — CHANGES_REQUESTED** [review]

**Comment by cloudthepain** [comment]
Looks good

File: IntroToCSharp/Shared/Components/Activities/DataTable.razor (hunk 1 above)
Comment (cwarren5701) [reply IntroToCSharp/Shared/Components/Activities/DataTable.razor:33]: Done

**Review by jcollard — APPROVED** [review]
lgtm

**Pull request merged**